proxy/logging/LogUtils.cc

```diff
@@ -599,7 +599,7 @@ LogUtils::file_is_writeable(const char *full_filename, off_t *size_bytes, bool *
     if (e < 0) {
       ret_val = -1;
     } else {
-      if (limit_data.rlim_cur != static_cast<rlim_t> RLIM_INFINITY) {
+      if (limit_data.rlim_cur != static_cast<rlim_t>(RLIM_INFINITY)) {
```

Comment:
Thanks, this is good code cleanup in general.  I think the only reason it hasn't broken for x86 build is that RLIM_INFINITY is implemented as macro with parentheses enclosing the value:
```
# define RLIM_INFINITY ((__rlim_t) -1)
```

Reply:
Ah, thanks @ywkaras. I was wondering how this built before.